core/services/keeper/registry_synchronizer_process_logs.go

```diff
@@ -125,7 +125,7 @@ func (rs *RegistrySynchronizer) HandleUpkeepRegistered(broadcast log.Broadcast,
 		return
 	}
 
-	err = rs.syncUpkeep(registry, utils.NewBig(upkeepID))
+	err = rs.syncUpkeep(&rs.registryWrapper, registry, utils.NewBig(upkeepID))
```

Comment:
I don't think we should pass members of `rs` as arguments to a function on `rs`

Reply:
It IS an antipattern, but the reasoning is described in another comment below. I did it this way to keep the scope of changes small while starting a path toward better testing coverage.